### PR TITLE
test-e2e/ansible : using samples

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
 Use Memcached sample instead of re-generate the project.

**Motivation for the change:**

operator-framework/enhancements#47

### CODE REVIEW

File: test/e2e-ansible/e2e_ansible_suite_test.go [hunk 1]
```diff
@@ -168,6 +139,38 @@ var _ = AfterSuite(func() {
...
+            - name: memcached
+              command:
+              - memcached
+              - -m=64
+              - -o
+              - modern
+              - -v
+              image: "docker.io/memcached:1.4.36-alpine"
+              ports:
+                - containerPort: 11211`
```
Comment (joelanford) [inline test/e2e-ansible/e2e_ansible_suite_test.go:172]: This looks nearly identical to what `memcachedWithBlackListTask` contains for the main deployment task. Is there a way to tweak things such that we just append the extra blacklist tasks to the existing sample without touching the deployment task?

If we replace this, we open ourselves to the possibility that we change the replacement substantially enough that the original sample is no longer being tested, and that would partially defeat the purpose of using the sample in the e2e test.

**Review by joelanford — COMMENTED** [review]
Looks great overall! Just one suggestion.

**Review by jmrodri — APPROVED** [review]
/lgtm
address joe's comment

File: test/e2e-ansible/e2e_ansible_suite_test.go (hunk 1 above)
Comment (camilamacedo86) [reply test/e2e-ansible/e2e_ansible_suite_test.go]: Sure, it is a good suggestion. Done. 👍

**Review by joelanford — APPROVED** [review]
/lgtm

**Pull request merged**